### DISCUSSION TITLE
Let us choose a game idea :)

### DISCUSSION THREAD

**Discussion by SilenLoc** [Plain-Raw/.github#1] [General]
@JeromeBollinger I think four ideas is enough. Let us choose.

**Comment by SilenLoc** [comment]
I am for the underground tower def / strategy game.

I think we can create something with bevy. But you may suggest a different engine. Keep in mind, we can use level editors also with bevy though.

**Reply by SilenLoc** [reply]
okay so underground tower def / strategy game. it is

**Reply by SilenLoc** [reply]
now a name

**Reply by JeromeBollinger** [reply]
Let's first think about a theme and then decide a name, otherwise our choice might be too generic.
